sound/soc/sof/sof-pci-dev.c

```diff
@@ -35,8 +35,28 @@ static int sof_pci_debug;
 module_param_named(sof_pci_debug, sof_pci_debug, int, 0444);
 MODULE_PARM_DESC(sof_pci_debug, "SOF PCI debug options (0x0 all off)");
 
+const char *sof_override_tplg_name;
```

Comment:
Oh, sorry for missing this during the first review rounds, I was thinking about getting rid of this global variable and missed to check it properly: this must be `static` of course.

Reply:
np, thank you. I made it static now and updated patch.